nemo_skills/inference/generate.py

```
@@ -634,6 +641,12 @@ async def async_loop(self, data):
         with open(self.cfg.output_file + "-async", "at", encoding="utf-8", buffering=1) as fout:
             # Dump prefilled data first
             if len(prefilled_data_points) > 0:
+                for output, data_point in zip(prefilled_outputs, prefilled_data_points):
```

Ruff (0.14.0)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)